Snippets/Snippets_3/DataBus/CleanupStrategy/Define.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Snippets3.DataBus.CleanupStrategy
+{
+    using NServiceBus;
+
+    public class Define
+    {
+        public Define()
+        {
+            #region DefineFileLocationForDatabusFiles
+            Configure.With()
```

Comment:
wouldnt this snippet suffice https://github.com/Particular/docs.particular.net/blob/master/Snippets/Snippets_4/DataBus/FileShare/DataBus.cs